src/index.html

```diff
@@ -11,8 +11,65 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="imac image"
+        class="card__image"
+      />
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <h4 class="card__code">Product code: 195434</h4>
+
+      <div class="card__rating">
+        <div class="stars">
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+        </div>
+
+        <h3 class="card__reviews">Reviews: 5</h3>
+      </div>
+
+      <div class="card__price-block">
+        <h3 class="card__price-title">Price:</h3>
+
+        <h1 class="card__price-amount">$2,199</h1>
+      </div>
+
+      <button class="card__buy-button">
```

Comment:
The button element should not contain the anchor `<a>` element. Instead, apply the `card__buy-button` class directly to the anchor tag and remove the button element. This is because semantically, a button should not be used to navigate, which is the purpose of an anchor tag.